dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/NacosNamingServiceWrapper.java

```diff
@@ -278,7 +278,7 @@ public ListView<String> getServicesOfServer(int pageNo, int pageSize, String gro
     }
 
     public List<Instance> selectInstances(String serviceName, String group, boolean healthy) throws NacosException {
-        return apply(() -> nacosConnectionManager.getNamingService().selectInstances(handleInnerSymbol(serviceName), group, healthy));
+        return apply(() -> nacosConnectionManager.getNamingService().selectInstances(handleInnerSymbol(serviceName), group, healthy, false));
```

Comment:
What is the purpose of change this? Dubbo will subscribe to that service after select. So `subscribe` can reuse the reqeust in one RT.

Reply:
It does't matter, i will close